.github/workflows/ci.yml

```diff
@@ -167,11 +127,10 @@ jobs:
         cargo test --all-features --workspace
         cargo test --package cuprate-blockchain --no-default-features --features redb
 
+    - name: Build
+      run: cargo build --all-features --all-targets --workspace
```

Comment:
Just for sanity checking, I'm planning to create a different workflow file for release builds + uploads.